src/options.rs

```diff
@@ -498,7 +500,7 @@ pub fn get_widget_layout(
 }
 
 fn get_update_rate_in_milliseconds(matches: &ArgMatches, config: &Config) -> error::Result<u64> {
-    let update_rate_in_milliseconds = if let Some(update_rate) = matches.value_of("rate") {
+    let update_rate_in_milliseconds = if let Some(update_rate) = matches.get_one::<String>("rate") {
```

Comment:
Hm, wonder why it's showing that you had to make a change here (and the rest of this file for any clap issues). If you want, it might be nice to rebase and clean up the commit history (e.g. those weird battery commits).